server.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const path = require('path');
+const dotenv = require('dotenv');
+
+dotenv.config();
+
+const app = express();
+const port = process.env.PORT || 7000;
+
+app.use(express.static(path.join(__dirname, 'src')));
+
+app.get('*', (req, res, next) => {
+  res.sendFile(path.join(__dirname, 'src/index.html'));
+});
+
+
+app.listen(port, () => {
```

Comment:
The callback argument passed to this method (app.listen) can also receive an `error` argument which would signify if an error occurred when starting your server. You can check this out.